### PR TITLE
convert last objects to FLU

### PR DESCRIPTION
**Pull request by joachimhgg**
- [x] (132) objects/robotstadium/protos/RobotstadiumSoccerField.proto
- [x] (133) objects/robotstadium/protos/RobotstadiumGoal.proto
- [x] (134) objects/lego/protos/LegoLargeMotor.proto
- [x] (135) objects/lego/protos/LegoWheel.proto
- [x] (136) objects/create_wall/protos/CreateWall.proto
- [x] (190) objects/backgrounds/protos/TexturedBackgroundLight.proto
- [x] (191) objects/backgrounds/protos/TexturedBackground.proto
- [x] (195) objects/freight/protos/IntermodalOfficeContainer.proto
- [x] (196) objects/freight/protos/IntermodalContainer.proto
- [x] (261) objects/lights/protos/DoubleFluorescentLamp.proto
- [x] (262) objects/lights/protos/ConstructionLamp.proto
- [x] (263) objects/lights/protos/CeilingSpotLight.proto
- [x] (264) objects/lights/protos/CeilingLight.proto
- [x] (265) objects/lights/protos/FloorLight.proto
- [x] (266) objects/floors/protos/CircleArena.proto
- [x] (270) objects/telephone/protos/OfficeTelephone.proto
- [x] (271) objects/telephone/protos/Telephone.proto
- [x] (282) objects/gear/protos/Gear.proto

### CODE REVIEW

File: projects/objects/lego/protos/LegoWheel.proto [hunk 1]
```diff
@@ -40,7 +40,7 @@ PROTO LegoWheel [
         geometry IndexedFaceSet {
```
Comment (BenjaminDeleze) [inline projects/objects/lego/protos/LegoWheel.proto:40]: The translation at line 7 need to be converted

File: projects/objects/freight/protos/IntermodalOfficeContainer.proto [hunk 2]
```diff
@@ -1,12 +1,12 @@
-#VRML_SIM R2021c utf8
+#VRML_SIM R2021b utf8
```
Comment (BenjaminDeleze) [inline projects/objects/freight/protos/IntermodalOfficeContainer.proto:1]:
```suggestion
#VRML_SIM R2021c utf8
```

File: projects/objects/freight/protos/IntermodalOfficeContainer.proto [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 # license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
 # license url: https://cyberbotics.com/webots_assets_license
 # An intermodal container is a large standardized shipping container, designed and built for intermodal freight transport, meaning these containers can be used across different modes of transport – from ship to rail to truck – without unloading and reloading their cargo.
 # An intermodal container with an office room.
 
 PROTO IntermodalOfficeContainer [
   field SFVec3f    translation    0 0 0        # Is `Transform.translation`.
-  field SFRotation rotation       0 0 1 0      # Is `Transform.rotation`.
+  field SFRotation rotation       0 1 0 0      # Is `Transform.rotation`.
```
Comment (BenjaminDeleze) [inline projects/objects/freight/protos/IntermodalOfficeContainer.proto:9]:
```suggestion
  field SFRotation rotation       0 0 1 0      # Is `Transform.rotation`.
```

File: projects/objects/freight/protos/IntermodalOfficeContainer.proto [hunk 4]
```diff
@@ -109,12 +109,12 @@ PROTO IntermodalOfficeContainer [
       }
     ]
     boundingObject Transform {
-      translation 0 1.3 0
+      translation 0 0 1.3
       children [
         Box {
-          size 12.192 2.6 2.438
+          size 12.192 2.438 2.6
```
Comment (BenjaminDeleze) [inline projects/objects/freight/protos/IntermodalOfficeContainer.proto:115]:
```suggestion
          size 12.192 2.438 2.6
```

File: projects/objects/lights/protos/ConstructionLamp.proto [hunk 5]
```diff
@@ -94,10 +94,10 @@ PROTO ConstructionLamp [
       }
```
Comment (BenjaminDeleze) [inline projects/objects/lights/protos/ConstructionLamp.proto:94]: The SpotLight need to be converted (View ->Optional Rendering -> Show light position can help)

File: projects/objects/lights/protos/CeilingLight.proto [hunk 6]
```diff
@@ -4,7 +4,7 @@
 # A ceiling light (0.19 x 0.8 x 0.19 m).
```
Comment (BenjaminDeleze) [inline projects/objects/lights/protos/CeilingLight.proto:4]:
```suggestion
# A ceiling light (0.19 x 0.19 x 0.8 m).
```

File: projects/objects/lights/protos/FloorLight.proto [hunk 7]
```diff
@@ -23,7 +23,7 @@ Solid {
   rotation IS rotation
```
Comment (BenjaminDeleze) [inline projects/objects/lights/protos/FloorLight.proto:23]: The description, line 4, need to be updated

File: projects/objects/floors/protos/CircleArena.proto [hunk 8]
```diff
@@ -90,7 +90,7 @@ PROTO CircleArena [
           coord Coordinate {
```
Comment (BenjaminDeleze) [inline projects/objects/floors/protos/CircleArena.proto:90]: The translation at line 78 need to be changed

File: projects/objects/telephone/protos/OfficeTelephone.proto [hunk 9]
```diff
@@ -1,12 +1,12 @@
-#VRML_SIM R2021c utf8
+#VRML_SIM R2021b utf8
```
Comment (BenjaminDeleze) [inline projects/objects/telephone/protos/OfficeTelephone.proto:1]:
```suggestion
#VRML_SIM R2021c utf8
```

File: projects/objects/telephone/protos/OfficeTelephone.proto [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
 # license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
 # license url: https://cyberbotics.com/webots_assets_license
 # An office telephone with optional physics.
 # template language: javascript
 
 PROTO OfficeTelephone [
   field SFVec3f    translation    0 0 0
-  field SFRotation rotation       0 0 1 0
+  field SFRotation rotation       0 1 0 0
```
Comment (BenjaminDeleze) [inline projects/objects/telephone/protos/OfficeTelephone.proto:9]:
```suggestion
  field SFRotation rotation       0 0 1 0
```

File: projects/objects/gear/protos/Gear.proto [hunk 11]
```diff
@@ -99,21 +99,21 @@ PROTO Gear [
                     sinAlpha = Math.sin(alpha);
                 >%
                   # base face
```
Comment (BenjaminDeleze) [inline projects/objects/gear/protos/Gear.proto:101]: Something seems wrong, the gear is flat

File: projects/objects/lego/protos/LegoWheel.proto [hunk 12]
```diff
@@ -4,7 +4,7 @@
 # LEGO Mindstrom EV 3 wheel. Parts id: 4634091 & 6035364.
 
 PROTO LegoWheel [
-  field SFVec3f    translation 0 0.0219974 0
+  field SFVec3f    translation 0 0 -0.9780026
```
Comment (BenjaminDeleze) [inline projects/objects/lego/protos/LegoWheel.proto:7]:
```suggestion
  field SFVec3f    translation 0 0 0.0219974
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Thank you, all comments have been addressed.

However, the gear seems still wrong (the texture is 'inside')

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Thank you, the gear looks nicer now.
However the bounding object with the option `teethBoundingObject` is wrong 
and the axis of the gear should be align with x (it is currently align with z)

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
It is better, but I managed to break the `teethBoundingObject` by putting the `toothDepth` to 0.03 or the `pitchRadius` to 0.05.

**Comment by ad-daniel** [comment]
- I believe the `IntermodalOfficeContainer` was backwards (i.e the door wasn't on the front as is the case for all buildings), so I fixed that and since I was on it I've collapsed all transforms (`IntermodalContainer` included).
- Fixed  the gear.
- The Cylinders/Capsules/Plane of `RobocupStadium` appear to be have been forgotten (probably because it's a PROTO that was already FLU), so I added it to this PR

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Pull request merged**